chrome/skin/style-windows.css

```diff
@@ -0,0 +1,27 @@
+#universal-search-recommendation {
+  background-color: Highlight;
+  color: HighlightText;
+}
+
+%ifdef XP_WIN
+@media (-moz-os-version: windows-vista) and (-moz-windows-default-theme),
+       (-moz-os-version: windows-win7) and (-moz-windows-default-theme) {
+  .autocomplete-richlistitem[selected="true"] {
```

Comment:
Replace `.autocomplete-richlistitem[selected="true"]` with `#universal-search-recommendation.highlight`, so that these styles are applied to our recommendation, not the autocomplete list items.